cortex_xdr_client/api/models/base.py

```diff
@@ -0,0 +1,9 @@
+
+
+from pydantic import BaseModel, Extra
+
+# Define a global base class with custom configuration
+class CustomBaseModel(BaseModel):
+    class Config:
+        extra = Extra.allow
```

Comment:
this allows us to get the whole dict used to create the pydentic object (missing keys)